port/common/omrport.tdf

```diff
@@ -701,12 +701,6 @@ TraceException=Trc_PRT_vmem_port_numa_set_affinity_node_not_present Group=mem Ov
 
 TraceEvent=Trc_PRT_vmem_port_numa_numa_support_could_not_be_configured_at_startup Group=mem Overhead=1 Level=1 NoEnv Template="omrvmem port_numa_interleave_memory: NUMA not available on this platform"
 
-TraceEntry=Trc_PRT_vmem_default_reserve_entry Group=mem Overhead=1 Level=5 NoEnv Template="default_pageSize_reserve_memory requestedAddress=%p byteAmount=%u"
-TraceEvent=Trc_PRT_vmem_default_reserve_using_mem_allocate_memory Group=mem Overhead=1 Level=5 NoEnv Template="default_pageSize_reserve_memory using omrmem_allocate_memory byteAmount=%u"
-TraceEvent=Trc_PRT_vmem_default_reserve_using_mem_allocate_memory32 Group=mem Overhead=1 Level=5 NoEnv Template="default_pageSize_reserve_memory using omrmem_allocate_memory32 byteAmount=%u"
-TraceException=Trc_PRT_vmem_default_reserve_failed Group=mem Overhead=1 Level=1 NoEnv Template="default_pageSize_reserve_memory FAILED requestedAddress=%p byteAmount=%u"
-TraceExit=Trc_PRT_vmem_default_reserve_exit Group=mem Overhead=1 Level=5 NoEnv Template="default_pageSize_reserve_memory returns %p requestedAddress=%p byteAmount=%u"
-
```

Comment:
I am not sure we can delete lines from trace file because handling is dependable for line number (other tools might include this file and should be backward compatible). The only way is use `Obsolete` key. @amicic is it correct?

Reply:
Right. Besides, could these really be removed (obsoleted) - aren't they used in some other platforms like AIX etc.?

Reply:
These are actually marked as obsoleted here`runtime/port/common/j9prt.tdf` in OpenJ9 side line 666. I should put them back in correct?

Reply:
They had been obsoleted probably when code moved from OpenJ9 to OMR. This is not so important for us here.

More important is that those trace points are still used in other platforms (I checked only first one, but could do for others), so they can't be removed. For Linux they may not make sense anymore, since methods changed names, so it's fine to introduce new ones based on old ones.

Reply:
That being said, I don't think you need to introduce new variants of Trc_PRT_vmem_default_reserve_using_mem_allocate_memory (and 32). They are specific to ZOS.